File-Formats/PDF/Working-with-forms.md

```diff
@@ -229,266 +154,162 @@ else
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Forms/Add-a-textbox-field-to-a-new-PDF-document).
 
-The below code snippet illustrates how to add the textbox to an existing PDF document.
+Load an existing PDF document using [PdfLoadedDocument](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedDocument.html) class, to add the textbox using [PdfTextBoxField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Interactive.PdfTextBoxField.html) class. The following code snippet explain this.
```

Comment:
Sentence formation is wrong... correct it as like the existing one

Reply:
changed the content